src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40; // Base cost per day is $40
```

Comment:
What is 40? Try to avoid magic numbers. Also, let's make totalCost a 'const' instead of 'let'